sale_invoice_policy/tests/test_sale_invoice_policy.py

```diff
@@ -36,6 +39,9 @@ def test_sale_order_invoice_order(self):
                 "invoice_policy": "order",
             }
         )
+        so._compute_invoice_policy_required()
```

Comment:
Why do you need to manually call the compute method? It shouldn't be needed.

Reply:
I'm not really sure, because there is code that comes from the old PR, but I can review it